power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -23,6 +23,7 @@ using TrafoGraphIdx = Idx;
 using EdgeWeight = int64_t;
 using WeightedTrafo = std::pair<Idx2D, EdgeWeight>;
 using WeightedTrafoList = std::vector<WeightedTrafo>;
+using RankedTransformerGroups = std::vector<std::vector<Idx2D>>;
 const int infty = INT_MAX;
```

Comment:
This change is not strictly necessary in our case here but indeed nice to have.

Reply:
well, the code would break if we ever decide to make `Idx` a `size_t` or a `long long` or whatever

Reply:
"break" in what sense? It is only used to initialize to a large than necessary value, which is unlikely to be exceeded in most systems. I could comfortably retire if I have `2147483647` even in JPY.

Reply:
wraparounds are easy to create when people write scripts. since we support things like negative IDs etc. it's better to be safe than sorry. you never know who copy-pastes your code ;D